doc/sphinx-guides/source/admin/metadatacustomization.rst

```diff
@@ -37,8 +37,8 @@ tab-separated value (TSV). [1]_\ :sup:`,`\  [2]_ While it is technically
 possible to define more than one metadata block in a TSV file, it is
 good organizational practice to define only one in each file.
 
-The metadata block TSVs shipped with the Dataverse Software are in `/tree/develop/scripts/api/data/metadatablocks
-<https://github.com/IQSS/dataverse/tree/develop/scripts/api/data/metadatablocks>`__ and the corresponding ResourceBundle property files `/tree/develop/src/main/java <https://github.com/IQSS/dataverse/tree/develop/src/main/java>`__ of the Dataverse Software GitHub repo. Human-readable copies are available in `this Google Sheets
+The metadata block TSVs shipped with the Dataverse Software are in `/src/scripts/api/data/metadatablocks
```

Comment:
They're in /scripts, not /src/scripts

Reply:
I made a PR against this branch to make this edit and one other wording change. Feel free to accept, decline, or change further.

Reply:
I added your changes! Thanks @qqmyers 🎉